api/receiver/receive_results.go

```diff
@@ -60,58 +59,58 @@ func HandleResultsUpload(a AppEngineAPI, w http.ResponseWriter, r *http.Request)
 		}
 	}
 
-	f, _, err := r.FormFile("result_file")
```

Comment:
@Hexcles - FYI - we were ignoring the error here, and `FormFile` actually calls `ParseMultipartForm` in effect.

Reply:
Now I'm really confused. `FormValue` also calls `ParseMultipartForm`. And we were not ignoring the error; it was checked later on L77. I don't see anything wrong yet.